src/openrct2/ride/VehicleEntry.h

```diff
@@ -72,4 +143,14 @@ struct rct_ride_entry_vehicle
     uint8_t pad_62[6] = {};
     std::vector<std::array<CoordsXY, 3>> peep_loading_waypoints = {};
     std::vector<int8_t> peep_loading_positions = {};
+
+    constexpr uint8_t GetNumRotationFrames() const
```

Comment:
As this is going to be used with image id calculation i suggest returning a uint32 cause you might perform unexpected truncation